src/views/List.jsx

```diff
@@ -17,17 +25,53 @@ export function List({ data, listToken }) {
 
 	// TODO - Use navigation similar to in Home.jsx to navigate to home page if no token or token is null
 	return (
-		<>
+		<Box display="flex" flexDirection="column">
 			{!data.length ? (
-				<div>
-					<h2>Hmmmm...looks like you have an empty list! 🤔 </h2>
-					<NavLink to="/add-item">
-						<button>Start List</button>
-					</NavLink>
-				</div>
+				addItem ? (
+					<AddItem data={data} setAddItem={setAddItem} />
+				) : (
+					<Paper
+						sx={{
+							py: 2,
+							px: 5,
+							display: 'flex',
+							flexDirection: 'column',
+							alignItems: 'center',
+							textAlign: 'center',
+						}}
+						elevation={3}
+					>
+						<Typography variant="h4" mb={2}>
+							🤔
+							<br />
+							Hmmmm...looks like you have an empty list!
+							<br />
```

Comment:
Haven't seen a ```<br >``` in a while. If you have the time and bandwidth, definitely try to find alternative way to style it. I believe  ```<br >``` aren't semantically appropriate. But I have heard some debate about it.